client/verta/verta/runtime.py

```diff
@@ -156,6 +160,8 @@ def get_embeddings(x):
         _validate_json(value)
     _validate_s3(key)
     local_logs: Dict[str, Any] = _get_thread_logs()
+    if key in local_logs.keys():
+        raise KeyError(f" cannot overwrite existing value for \"{key}\"")
```

Comment:
This might be a hot take, but I think this should be a `ValueError`.

Python's documentation describes `KeyError` exclusively as a lookup error (in fact, its base class is `LookupError`), rather than an assignment error:

> _exception_ **KeyError**
> Raised when a mapping (dictionary) key is not found in the set of existing keys.

Reply:
Done